src/json-patch-duplex.js

```diff
@@ -198,20 +199,18 @@ var jsonpatch;
         return '/' + path;
     }
     var beforeDict = [];
-    var Mirror = (function () {
-        function Mirror(obj) {
+    class Mirror {
```

Comment:
IE 11 does not support `class`. That's why we need to keep ES5-style class declarations, at least in js